zulipterminal/model.py

```diff
@@ -1447,7 +1447,13 @@ def notify_user(self, message: Message) -> str:
                 set(message["flags"])
             ) or self.is_visual_notifications_enabled(stream_id):
                 recipient = "{display_recipient} -> {subject}".format(**message)
-
+            if "has_alert_word" in message["flags"]:
+                # Check if stream or topic is muted
+                topic = message.get("subject", "")
+                if not self.is_muted_stream(stream_id) and not self.is_muted_topic(
+                    stream_id, topic
+                ):
+                    recipient = "{display_recipient} -> {subject}".format(**message)
```

Comment:
Could you address this, to avoid the duplication? The same action is done in each case, so we can combine the conditionals, though I'd suggest some intermediate variables to make the conditional more readable.